src/main/java/net/dv8tion/jda/api/entities/MessageReaction.java

```diff
@@ -528,6 +528,20 @@ public long getIdLong()
                 throw new IllegalStateException("Cannot get id for emoji reaction");
             return id;
         }
+        
+        /**
+         * The code for this Reaction.
+         * <br>For unicode emojis this will be the unicode of said emoji rather than an alias like {@code :smiley:}.
+         * <br>For custom emotes this will be the name and id of said emote in the format {@code <name>:<id>}.
+         *
+         * @return The unicode if it is an emoji, or the name and id in the format {@code <name>:<id>}
+         */
+        public String getAsReactionCode()
+        {
+            return isEmote()
+                    ? name + ":" + id
+                    : EncodingUtil.encodeUTF8(name);
```

Comment:
How do you expect this to be used? If you encode it here then it will encoding it twice if used with `addReaction`.

Reply:
I based it off the `getReactionCode` from the MessageReaction class, since this one had the EncodingUtil.encodeUTF8 method.
I'll remove it I suppose.